Analytics/Integrations/Taplytics/SEGTaplyticsIntegration.m

```diff
@@ -28,9 +28,17 @@ - (id)init {
 }
 
 - (void)start {
-  [Taplytics startTaplyticsAPIKey:[self apiKey]];
+  NSDictionary *options = [[NSMutableDictionary alloc] init];
   
-  SEGLog(@"TaplyticsIntegration initialized with api key %@", [self apiKey]);
+  if ([self delayLoad])
+      [options setValue:[self delayLoad] forKey:@"delayLoad"];
```

Comment:
We believe so. Some initial tests showed that it worked internally. As long as the users end up initialize Segment in the `didFinishLaunching:` method.

We'll be testing this with a few clients over time and will modify as we go if there's any issues.